conf/deployment/ibmcloud/ibm_cloud_install_odf_addon.yaml

```diff
@@ -0,0 +1,7 @@
+# This is the basic config for OCS deploy with addon
+---
+ENV_DATA:
+  storage_cluster: 'roks4.8'
+  platform: 'ibm_cloud'
+  osdSize: '250Gi'
```

Comment:
We are using this value:
device_size

But you should not set it in config. It's set via --osd-size CLI parameter